api_tests/lib/bitcoin.ts

```diff
@@ -0,0 +1,165 @@
+import {
+    Transaction,
+    ECPair,
+    Out,
+    payments,
+    networks,
+    TransactionBuilder,
+    address,
+} from "bitcoinjs-lib";
+const BitcoinRpcClient = require("bitcoin-core");
+const sb = require("satoshi-bitcoin");
+const util = require("./util.js");
+
+// TODO: Remove regtest
+// Once this is merged: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33571
+
+const networksAny: any = networks;
+const regtest: any = networksAny.regtest;
+
+interface IBitcoinConfig {
```

Comment:
Such is life.

Reply:
> This is how I have seen it done. However looks like in TypeScript doc they just do Capitalisation.

@D4nte It is originally a C# thing and TS as well as C# are made by the same company, Microsoft. That is why there are similar conventions. I don't like it either because IMO, it clutters names with a meaningless character that are only in your way when doing type searches etc.